tests/regression/29-svcomp/11_arithmetic_bot.c

```diff
@@ -0,0 +1,40 @@
+// PARAM: --conf=conf/svcomp21.json
```

Comment:
Might be better to have the few specific parameters added directly in the tests.

Reply:
Ok, done https://github.com/goblint/analyzer/pull/131/commits/b061cc03d3af59807ddc70773c510b81022f7d06